bandcamp_dl/__main__.py

```diff
@@ -30,6 +30,7 @@
     -a --ascii-only         Only allow ASCII chars (北京 (capital of china) -> bei-jing-capital-of-china)
     -k --keep-spaces        Retain whitespace in filenames
     -u --keep-upper         Retain uppercase letters in filenames
+    --no-confirm            Override confirmation prompts. Use with caution.
```

Comment:
That is quick & dirty. Could be elaborated / written more precicely what it is about. Suggestions welcome!